### PR TITLE
release pr/0.11b0

### PR DESCRIPTION
**Pull request by aabmass**
This creates two commits; please review them individually. The first commit is the released code, where I will add a release branch `release/0.11x` and tag `0.11.0` (trying to roughly follow opentelemetry-python's conventions). Any cherry picked commits will go on top of `release/0.11x`. The second commit is the new dev version that the repo will stay with until the next release.

I will merge this PR with "Rebase and Merge" so they both appear in master history. After merged, I will create a tag for the release (commit 1/2) and release pypi packages. I generated these commits with a script which I will add along with some instructions in a separate PR 🙂 

Commits:
- Release 0.11b0 (Part 1/2) release commit
  - Update version.py files
  - Marked releases in changelogs
  - Pinned `opentelemetry-{api,sdk}` versions in dev-constraints
  - Pinned `opentelemetry-{api,sdk}` versions in each package's `setup.cfg` file
- Release 0.11b0 (Part 2/2) bump version to 0.12.dev0
  - Update version.py files
  - Unpin `opentelemetry-{api,sdk}` versions in each package's `setup.cfg` file

### CODE REVIEW

File: opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/google/version.py [hunk 1]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.11.dev0"
+__version__ = "0.12.dev0"
```
Comment (AndrewAXue) [inline opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/google/version.py:15]: Shouldn't this be 0.11?

File: opentelemetry-tools-google-cloud/src/opentelemetry/tools/google/version.py [hunk 2]
```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.11.dev0"
+__version__ = "0.12.dev0"
```
Comment (AndrewAXue) [inline opentelemetry-tools-google-cloud/src/opentelemetry/tools/google/version.py:15]: Same here, shouldn't it be 0.11?

File: opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/google/version.py (hunk 1 above)
Comment (cnnradams) [reply opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/google/version.py:15]: the dev version is `0.12.dev0` (after the 2nd commit), the first commit does `0.11.b0` which is the right version for the release.

**Comment by aabmass** [comment]
Just reformatted the commit messages since they will end up on master

**Pull request merged**